MathlibTest/CountHeartbeatsLinter.lean

```diff
@@ -0,0 +1,24 @@
+import Mathlib.Util.CountHeartbeats
+
+-- sets the `countHeartbeats` linter option to `true`
+count_heartbeats
+
+mutual -- mutual declarations get ignored
+theorem XY : True := trivial
+end
+
+/-- info: Used 3 heartbeats, which is less than the current maximum of 200000. -/
+#guard_msgs in
+set_option linter.unusedTactic false in
+set_option linter.unusedTactic false in
```

Comment:
what's the point of the duplication? If I clone the file, it seems to work fine with just one of the two.

Reply:
I was just testing that `withSetOptionIn` crawled inside *all* `set_option`s, not just the outermost one.

I probably prefer to keep the repetition, even if it is not strictly a test of `count_heartbeats`.

Reply:
Maybe worth adding a comment if you keep it since it is confusing at first glance.

Reply:
Comment added, thanks for the suggestion!